### PR TITLE
Doc(UPM-3733): update Create Azure Active Directory Application

### PR DESCRIPTION
**Pull request by Simonzhaohui**
## What Changed?

Based the latest script [`create-spn.sh`](https://github.com/EnterpriseDB/cloud-utilities/blob/main/azure/create-spn.sh), modified file 02_connect_cloud_account.mdx to be used for content of
Connecting your BigAnimal account

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

File: product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx [hunk 1]
```diff
@@ -31,6 +31,7 @@ Set up your BigAnimal account on Azure Marketplace. Your Azure subscription for
 Take note of the **Application (client) ID**, as you need it to configure your BigAnimal account. Also take note of the **Display name** value of the Azure AD application. You need to enter this value later.
 
 1. Select _application secret_ as an authentication option for the application. See [Create a new Azure AD application secret](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#option-2-create-a-new-application-secret) for instructions. Take note of the Azure AD App Secret, as you need it to configure your cloud account.
+1. Select _API permissions_ to configure API permissions for the application. See [Configure a client application to access Azure Active Directory API](https://docs.microsoft.com/en-us/azure/active-directory/reports-monitoring/howto-configure-prerequisites-for-reporting-api) for instructions. Add Application permissions with Microsoft Graph _Application.ReadWrite.OwnedBy_ and _Directory.Read.All_ to your application, and grant admin consent for your cloud account.
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx:34]:
```suggestion
1. Select _API permissions_ to configure API permissions for the application. See [Configure a client application to access Azure Active Directory API](https://docs.microsoft.com/en-us/azure/active-directory/reports-monitoring/howto-configure-prerequisites-for-reporting-api) for instructions. Add Application permissions with Microsoft Graph _Application.ReadWrite.OwnedBy_ and _Directory.Read.All_ to your application and grant admin consent for your cloud account.
```

Comment (Simonzhaohui) [reply product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx]: Updated.

File: product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx [hunk 2]
```diff
@@ -62,22 +66,39 @@ The specific output includes the client ID and client secret you use when you co
...
+Waiting 15 seconds for Azure AD Service Principal to propagate...
+Add Azure AD Service Principal Owners...
 {
-  "client_id": "8d9fxxxx-xxxx-xxxx-xxxx-xxxxxxxx39f4",
+  "client_id": "a756xxxx-xxxx-xxxx-xxxx-xxxxxxxx84da",
   "client_secret": "xxxx",
   "subscription": "c808xxxx-xxxx-xxxx-xxxx-xxxxxxxxb712"
 }
 ```
+If you receive below error messages, you need to apply admin consent request with Azure AD Global Administrator or Privileged Role Administrator.
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx:97]:
```suggestion
If you receive the following error message, you need to request admin consent for your cloud account. Only users with the Azure AD Global Administrator or Privileged Role Administrator role can grant admin consent.
```

**Review by drothery-edb — APPROVED** [review]
LGTM - left two comments for your consideration.

**Comment by Simonzhaohui** [comment]
Hello @drothery-edb, thanks for your comments. I updated, pls help to approve again.

**Pull request merged**